docs/form-question-types.rst

```diff
@@ -1681,6 +1681,9 @@ Image widgets
 .. tip::
   Image files can be very large. We recommend always including :ref:`a maximum image size in form design <scaling-down-images>`. Also, consider making test submissions to your server with the Internet conditions you expect when gathering data to make sure that you can send files of the size you expect.
 
+.. warning::
+  As of Android 13, Google introduced a new photo picker, backported to devices running Android 11 and 12. This new picker is automatically used when users click the ``Choose Image`` button. The picker enhances privacy by removing sensitive metadata, including GPS location, from selected images. If you rely on metadata like GPS location, please note that this data may no longer be available.
```

Comment:
This reads as a bit more technical than it needs to be! Does it only affect images that are picked? Are GPS coordinates always included in pictures that are taken directly from Collect?

How about "Sensitive metadata, including GPS location, is generally no longer included in images selected from :guilabel:`Choose Image` on Android 11+." Then you won't need the "backported" addition to spelling either.

Reply:
> Does it only affect images that are picked?

Yes.

> Are GPS coordinates always included in pictures that are taken directly from Collect?

It depends on the camera app if it is supported then yes.

> How about "Sensitive metadata, including GPS location, is generally no longer included in images selected from :guilabel:Choose Image on Android 11+." Then you won't need the "backported" addition to spelling either.

Ok we can use such a simple message. I will update the pull request.